### PR TITLE
stolonctl: add page

### PR DESCRIPTION
**Pull request by VidhiG**
Stolon is a cloud native PostgreSQL manager for PostgreSQL high availability, and stolonctl is the command line tool for it.

Ref: https://github.com/sorintlab/stolon

### CODE REVIEW

File: pages/common/stolonctl.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Stolon
+
+> Stolon is a cloud native PostgreSQL manager for PostgreSQL high availability.
```
Comment (agnivade) [inline pages/common/stolonctl.md]: We can remove "Stolon is"

File: pages/common/stolonctl.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Get cluster status:
+
+`stolonctl --cluster-name ${{stolon_cluster_name}} --store-backend ${{stolon_store_backend}} --store-endpoints ${{stolon_store_endpoints}} status`
```
Comment (agnivade) [inline pages/common/stolonctl.md]: I think we can remove "stolon_" from the token values. It is clear enough that the items are about stolon.

File: pages/common/stolonctl.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- Get cluster data:
+
+`stolonctl --cluster-name ${{stolon_cluster_name}} --store-backend ${{stolon_store_backend}} --store-endpoints ${{stolon_store_endpoints}} clusterdata`
+
+- Get cluster specification:
+
+`stolonctl --cluster-name ${{stolon_cluster_name}} --store-backend ${{stolon_store_backend}} --store-endpoints ${{stolon_store_endpoints}} spec`
+
+- Update cluster specification:
```
Comment (agnivade) [inline pages/common/stolonctl.md]: "Update cluster specification with a patch in json format:"

File: pages/common/stolonctl.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+`stolonctl --cluster-name ${{stolon_cluster_name}} --store-backend ${{stolon_store_backend}} --store-endpoints ${{stolon_store_endpoints}} update --patch '{{cluster_spec_to_be_updated_as_json}}'`
```
Comment (agnivade) [inline pages/common/stolonctl.md]: s/cluster_spec_to_be_updated_as_json/cluster_spec

File: pages/common/stolonctl.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Stolon
+
+> A cloud native PostgreSQL manager for PostgreSQL high availability.
+
+- Get cluster status:
+
+`stolonctl --cluster-name ${{cluster_name}} --store-backend ${{store_backend}} --store-endpoints ${{store_endpoints}} status`
```
Comment (sbrl) [inline pages/common/stolonctl.md]: Are the dollar signs `$` are required when calling the command?

Comment (VidhiG) [reply pages/common/stolonctl.md]: @sbrl no they will be needed only if there is a variable substitution. Should I remove the `$` symbol?

Comment (agnivade) [reply pages/common/stolonctl.md]: Yes, please remove it.

Comment (VidhiG) [reply pages/common/stolonctl.md]: @agnivade @sbrl I removed it.

**Pull request merged**